### DISCUSSION TITLE
The realistic guide to presets.

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#140] [Presets]
# Guide to presets

I wanted to help people create realistic presets and learn some of my techniques, so I made this large guide. Enjoy!

Estimated Reading Time: `8 minutes, 44 seconds`

---

## Tip 1: Learn the settings.
I mean all of them. Figure out what they do, either by trial and error or documentation. This is **crucial**, as it can help you know which effect you want (and how to get it!).

## Tip 2: Get a VCR. (recommended, optional).
A VCR is very useful, as it can show you what a true VHS/Analog video signal looks like. Study the video signal and original picture/video next to each other. It helps to know what distortions and effects are in the signal.

## Tip 3: Look at some of my presets!
Yes, a small plug indeed, but they show you which effects I use, and their intensity. Useful for learning what effect should not be overused, or underused!

## Tip 4: Know what you're going after.
What effect are you trying to go for? Analog signals? Or could it be a worn out tape from 1993? Whatever it is, it helps to know what you are going after. Having a clear picture in your mind is crucial.

## Tip 5: Don't use modern applications as your video.
This is a big "don't" for me. For example: Modern games (Roblox, Minecraft, COD, etc.), full screen recordings of desktop (unless it is windows NT 2000 or lower, use digital recording with high compression) For a few reasons:

1. The video loses its authentic feel when the input is modern.
2. Modern games are designed for modern screens. Which mean smaller icons and such. `The Legend of Zelda - The - Ocarina of Time` is a perfect example of a good game to use this effect on, as it was made for the lower resolution, 1998 CRT screens. As a result, it has clearer, and larger UI to work with.
3. There isn't really a "modern tape" to go off of (see `Tip 7`).

## Tip 6: VHS and Analog signals weren't that distorted.
Indeed, it is fun to toy around with the distortions and damages, but tape wasn't really that glitchy or damaged. If you find an old commercial online and it looks noisy, there are a few reasons:

1. The user was far away from the TV transmitting station, resulting in a noisy input image (noise, small image jumps).
2. The digitizer/VCR had bad tracking (noise, snow, jumpy image).
3. The tape is damaged (noise, snow, extreme jumping, speed changes/audio speed changes).

Other than those, there isn't really much noise on a tape.

## Tip 7: Mimic a tape from the year the input video is supposed to be.

This is also crucial and sort of ties into `Tip 4`. Lets take some examples here:

Lets say your input video is `Super Mario Bros.` for the NES. A classic that came out in 1985. There aren't many videos I could find of normal consumer VHS tapes with games in 1985, but the ones that do are mostly 1" open reel tapes. Now that we know this, we can simulate the tape. For open reel tapes, these usually have the following characteristics:

- Blurry luminance
- No sharpening
- Box filtering (input, demod)
- Some noise (not too much)
- A hint of snow
- Chroma smearing (left + right)
- Head switching
- Small amount of tracking noise

But wait, what about the other years? Lets take 1998's Ocarina of Time for the N64. There are plenty of videos online of consumer VHS tapes used to record this game. These tapes are usually digitized by armatures with normal VCRs. Lets look at the characteristics:

- Sharper image
- Middle tier sharpening
- Notch Filtering demod
- No input filter or Notch
- Some noise
- Chromatic noise
- Higher bandwidth
- Head switching
- Small amount of tracking noise

You can look at other types of tapes and eras at [AVAA](https://www.avartifactatlas.com)

## Tip 8: AVAA
Speaking of AVAA, that site is an amazing source for artifacts and specifications. **HIGHLY** recommend going there as well after this guide.

## Tip 9: Modify your own presets to make a new one!
Once you make a preset, you can use that one as a base instead of starting over. Granted you know what you are going for, that is.

## Tip 10: Use real life footage from an old digital camera (HIGHLY RECOMMENDED FOR REAL FOOTAGE)
I can't stress this enough, use a real camera from around 2001 or 2006 (must be digital). Why? well they use the same sensors as true VHS cameras, and they are light and inexpensive. 

Huge thanks to @notdigidani for a good camera recommendation. 

>and to answer your question, ive used my KODAK PIXPRO FZ101 to record the video. no problem!

Full conversation [here](https://github.com/valadaptive/ntsc-rs/discussions/132).

## Tip 11: Compression is key to a good video.
On to exporting settings, I recommend H.264 with a quality of 20, and a speed of 5.

Now this really depends on the purpose. Just like `Tip 7`, where there are the archiver and consumer tapes. Because archival sites for tapes have to store so many, the video is compressed to save space in the servers. If it is a consumer, however, most of the time the video will have lower compression.

I personally enjoy the "archival" look, which is actually mostly comprised of open-reels. but its more personal preference.

<details>
<summary>Examples</summary>

Compressed:

![image](https://github.com/user-attachments/assets/69ede7f0-dca9-4d0f-9de1-460d6c3b5b2e)

Uncompressed:

![image](https://github.com/user-attachments/assets/9cef50ef-7b2c-44af-a57c-3c2e4c87becc)

</details>

## Tip 12: OPEN REEL ≠ VHS

Open reel and VHS are **entirely separate things!** 

VHS stands for Video Home System. JVC released the first VHS machines in Japan in late 1976, and in the United States in mid-1977. Sony's Betamax, JVC's VHS competitor, is a little older. Sony Betamax made its debut in the late 1970s. They both look visually similar when digitized, but Betamax and VHS are separate.

Open reel tape, created much before VHS in 1951, was a very popular successor to film. It did not offer much quality however which is why digitized videos from then are on film. This is why some 50s shows have the "VHS look"... well, not really. Betamax, Open Reel, and VHS are all tape based, making it a "Tape look". 

### Now why does this matter?

If you want to make a "VHS preset", odds are you are not going to make a very unique preset, as tapes mostly look the same (if under the same condition). Experimenting with other tape formats and knowing what they look like can very much help the realism, as not all tape formats are "VHS" - VHS is just a popular tape format.

### What's the difference in quality?

Well, since Open Reel is older than VHS, it has a little worse quality and some color artifacts.

<details><summary>1/2" OPEN REEL</summary>
<p>

https://github.com/user-attachments/assets/b98331ab-7833-45c8-8fba-eca3f3d967c5

</p>
</details> 

<details><summary>VHS</summary>
<p>

https://github.com/user-attachments/assets/2dff620c-1f55-44c9-a731-6171eab66e84

</p>
</details> 

---

# Special guide for AV digitizer presets

I'm putting this in its own section, since tape presets are a LOT different than AV. RCA or AV signals all have their distinct looks. This guide will help you get the perfect look.

**NOTES**: 

## Every RCA cable has its own look.
As I said in the intro, they dont always look the same. Some have combing filters, some have lower bandwidth, so you can't really make a "universal" preset or a "capture card" preset.

---

## AV Tip 1: Use a real source to analyze.
Just like getting your own VCR, digitize the output of an analog source. For this guide however, I will be using the Sega Genesis, or Megadrive, as the output.

## AV Tip 2: Settings to avoid, and settings to limit.
This is important. Here are the settings to avoid:

| Name   | Type   | Reason |
|--------|--------|--------|
| VHS | Avoid | Only use in special circumstances, like ATV or Black and White filtering |
| Composite Noise | Required, Limit | This is a must have for AV, but should be used in moderation. |
| Bandwidth | Limit | Remember that a typical RCA jack has a bandwidth of 5Mhz (around 1.5 for NTSC-RS) |
| Chromatic Phase Noise | Extreme limiting | It can really mess up the look and feel of the preset and output. | 
| Ringing | Slight limit | Ringing is very useful, but the power should not be intense, and the scale should be low so it does not get in the way. The frequency should be around 50% - 85% | 
| Constant K | Avoid | AV is sharp. Avoid this blurry lowpass. |
| Head switching/Tracking noise | Not used | Never use these. These effects are only visible on tape from a VCR or player. | 

## AV Tip 3: Demodulation
Very important. RCA uses Comb demodulation most of the time. The Sega Genesis has really intense comb lines. Here is a screenshot from my digitization for a preset

![Screenshot 2024-07-24 07-58-27](https://github.com/user-attachments/assets/5f0dabf1-1485-46c2-9976-cf6e007ed183)

>[!NOTE]
> Notice the lines on the tops and bottoms of edges. This is comb demodulation at work. Most likely, `2 Line Comb`

With that reference image, we can see how the demodulation works (FOR THE GENESIS)

>[!WARNING]
>I say that this is for the Genesis, because that is just how the RCA cable for the genesis is designed, and might not be the same for say, the N64.

## AV Tip 4: Bandwidth
The average RCA video signal bandwidth is around 5Mhz. NTSC-RS's bandwidth signal is around around -1:0.3 or 1Mhz per 0.3 NTSC-RS bandwidth value. This gives us a value of around 1.5 for our 5Mhz signal. (Could be 2 if you want)

## AV Tip 5: Chroma delay
Tying into `AV Tip 4`, The lower the bandwidth the more the effects are strengthened. There is a chroma delay built in to the effects, but if you add a delay to the left via the `Horizontal Chroma Delay`, you can balance them out, making the chrominance become "smeared". Figure out a good balance for 1.5 or 2 bandwidth.

## AV Tip 6: The nitty, gritty details
These are minor details, but they serve a good purpose

| Name | Limit | Details |
|--------|--------|--------|
| Chrominance noise | Limited, Almost unnoticeable. | This provides a slight noise in the colors, and should be a high detail, high frequency, but light effect.|
| Composite noise | Moderate limitation. | This is very important as well, but used subtly. Should be a lower frequency (until you start seeing color aliasing) and low detailed. |
| Luma Smear | Extreme moderation. | Only up to 0.2 is recommended|
| Input filtering | Depending on goal | This is mostly used with Box, but "none" is also pretty realistic in some cases |

# Final notes.
Yes. This guide was very long! I hope you find some use with it. This guide will be added on to or updated in the future. If you make anything using this guide, please add credit to this guide for others to learn!

All images created by me.

# Updates:

7/30/2024: Added `Tip 12`, fixed grammatical errors and inconsistencies

**Comment by ntsc-rs-fan** [comment]
Which one is Bandwidth on web.ntsc.rs? The scale? Or is it there?
